src/Velociraptor.js

```diff
@@ -13,25 +13,24 @@ module.exports = React.createClass({
     displayName: 'Velociraptor',
 
     propTypes: {
-        children: React.PropTypes.array,
+        // either a list of child nodes, or a single child.
+        children: React.PropTypes.any,
+
+        // optional transition information for changing existing children
         transition: React.PropTypes.object,
-        container: React.PropTypes.string
     },
 
     componentWillMount: function() {
         this.children = {};
         this.nodes = {};
-        var self = this;
-        React.Children.map(this.props.children, function(c) {
-            self.children[c.key] = c;
-        });
+        this.componentWillReceiveProps(this.props);
```

Comment:
Maybe just one for the future but I've tried to avoid calling react interface methods from other interface methods as it can be confusing, and instead make a common function that's called from both places.